docs/clients.md

```diff
@@ -0,0 +1,21 @@
+# RedisTimeSeries Client Libraries
+
+RedisTimeSeries has several client libraries, written by the module authors and community members - abstracting the API in different programming languages. 
+
+While it is possible and simple to use the raw Redis commands API, in most cases it's easier to just use a client library abstracting it. 
```

Comment:
@filipecosta90 "it's more convenient"

Reply:
Fixed in the latest commit. Please revise